### PR TITLE
Enable Spatial Parameters

### PR DESCRIPTION
**Pull request by Ukjain**
This will enable spatial parameters to be added in the Solr query

### CODE REVIEW

File: src/main/scala/com/github/takezoe/solr/scala/QueryBuilderBase.scala [hunk 1]
```diff
@@ -281,4 +281,37 @@ trait QueryBuilderBase[Repr <: QueryBuilderBase[Repr]] {
     )
   }
 
+
+  /**
+   * Added in response to #63 Missing spatial parameters while building solr query
+   * Sets Spatial field called pt.
+   * @param value : value for point , typically represent lat,long
+   */
+  def spatialPt(value: String): Repr = {
```
Comment (takezoe) [inline src/main/scala/com/github/takezoe/solr/scala/QueryBuilderBase.scala]: `def spatialPt(lat: Double, lon: Double)` might be a better interface. What do you think?

Comment (Ukjain) [reply src/main/scala/com/github/takezoe/solr/scala/QueryBuilderBase.scala]: i agree . Raising new pull request

**Comment by Ukjain** [comment]
Please review it again. I have changed the method interface

**Comment by Ukjain** [comment]
Hey can we make it fast

**Comment by takezoe** [comment]
Looks good! Thanks!

**Pull request merged**